variables.tf

```diff
@@ -3,139 +3,169 @@
 # ------------------------------------------------------------------------------
 
 variable "apply_immediately" {
-  type        = "string"
   description = "Specifies whether any database modifications are applied immediately, or during the next maintenance window"
+  type        = string
```

Comment:
Ideally we'd just use proper types as part of the upgrade. So this one should be `bool` and default to `false` instead of `"false"`?

Reply:
I was not sure if the input is string and value is "false" as an arbitrary string, or it is a real bool value, thanks! :-)